### DISCUSSION TITLE
Similar feature to "close when empty" in VSCode

### DISCUSSION THREAD

**Discussion by dabeeeenster** [zed-industries/zed#16530] [Feature Requests]
When there are no tabs open in the editor, it would be nice to have an option to prevent the window itself from closing if I hit cmd+w - VScode has an option "close when empty" that can prevent this - I cant find anything similar in Zed?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
